### ISSUE TITLE
Add toll-free phone number

### ISSUE THREAD

**Issue by Alcalt** [pods-framework/pods#2575]
Could you please add a way to <b>format phone number</b> or <b>add a toll-free field</b> on Pods 2.5.1? I'm from Canada and our toll-free phone number are too long by 1 or 2  number.  Pods still show them but not in the right format (ie: <b>180-050-612-34567</b> instead of <b>1-800-506-123-4567</b>).

I thank you in advance

**Comment by sc0ttkclark** [comment]
Are there any other formats you need?

**Comment by Alcalt** [comment]
Not really. A compatible version with WPML would be nice but I think you are planning to do it in you next major update ( or it's them I can't remember) so for now I'm good. The current version already have pretty much all I need.

**Comment by Alcalt** [comment]
Forget the <b>506</b> part. I forgot to remove it when I change to a toll-free number. The correct format is <b>1-800-123-4567</b>. It's still a error since US number have 10 number long and we have 11.
